topology/probes/ovs_of.go

```diff
@@ -494,6 +502,7 @@ func (o *OvsOfProbe) OnOvsBridgeAdd(bridgeNode *graph.Node) {
 		return
 	}
 	bridgeProbe, err := o.NewBridgeProbe(hostName, bridgeName, uuid, bridgeNode)
+	logging.GetLogger().Infof("Probe added for %s on %s", bridgeName, hostName)
```

Comment:
log the uuid as well

Reply:
Yes and made more consistent with error case.